CONTRIBUTING.rst

```diff
@@ -7,15 +7,20 @@ any help you give.
 
 When submitting a pull request, we ask you check the following:
 
-1. **Unit tests**, **documentation**, and **code style** are in order. 
+1. A **code of conduct**,
```

Comment:
I would put this as a separate paragraph or maybe at the end of the first paragraph, instead of adding it in this list --- the list is more of a technical checklist.

Along the lines of 
"""
Note that [the code of conduct](https://github.com/scipy/scipy/blob/master/doc/source/dev/conduct/code_of_conduct.rst) applies to all spaces managed by the SciPy project, including issues and pull requests.
"""